src/app/src/main/java/com/uphold/uphold_android_sdk/model/transaction/TransactionCommitRequest.java

```diff
@@ -29,14 +32,38 @@ public TransactionCommitRequest(String message) {
      */
 
     public TransactionCommitRequest(String message, String securityCode) {
+        this.beneficiary = null;
         this.message = message;
         this.securityCode = securityCode;
     }
 
+    /**
+     * Constructor.
+     *
+     * @param beneficiary The transaction beneficiary.
+     * @param message The transaction message.
+     */
+
+    public TransactionCommitRequest(Beneficiary beneficiary, String message) {
```

Comment:
I think we are missing the constructor with only the `Beneficiary`. Can you please verify?